controller/proxy-injector/webhook_test.go

```diff
@@ -79,30 +79,12 @@ func TestGetPatch(t *testing.T) {
 				conf:     confNsEnabled(),
 				expected: true,
 			},
-			{
-				filename: "deployment-inject-disabled.yaml",
-				ns:       nsEnabled,
-				conf:     confNsEnabled(),
-				expected: false,
-			},
-			{
-				filename: "deployment-inject-empty.yaml",
-				ns:       nsDisabled,
-				conf:     confNsDisabled(),
-				expected: false,
-			},
```

Comment:
Is it enough to add the additional `Injectable()` test below to be able to re-enable these tests, or is it more complicated than that? If so we could track that as a separate item for the 2.3 test sprint to expedite this PR :zap:

Reply:
These two test cases are no longer useful, as the results will always be `true` because `GetPatch()` no longer checks if injection should happen. Some of these tests need to be revisited, as they aren't testing anything webhook related, but not in this PR.